### PR TITLE
UI rework towards Contribution Hub

### PR DESCRIPTION
**Pull request by comzeradd**
UI reworked since this is now more than a FoxFooding platform. I removed the big sidebar; having essential content and specific 'call to action' we can get more focused user experience.

### CODE REVIEW

**Comment by comzeradd** [comment]
@akatsoulas r?

**Comment by akatsoulas** [comment]
screen shot 2015-10-14 at 12 40 27

A few thoughts/comments:
- Footer seems to be out of place.
- The fxos-wordmark seems too big to me.
- The same goes to homescreen. Plus it seems a bit funny since only half of the image is visible.
- Mobile is broken.

**Comment by akatsoulas** [comment]
The images are missing from the 'devices' section.

**Comment by akatsoulas** [comment]
I would suggest to leave the dashboard in place until decided otherwise.

**Comment by akatsoulas** [comment]
Please provide a better, more descriptive alt description to all images.

**Comment by comzeradd** [comment]
@akatsoulas PR updated

I did some html/css changes:
- Added dashboard as a navigation link
- Removed broken image from devices. We should add these when we know which devices.
- Added missing alt to some images
- Tweak a bit the footer, but without content it's not easy to fix it at the bottom for all resolutions.
- I kept the homescreen image for the tablet resolutions too (the one on your screenshot)

File: feedthefox/base/static/css/app.css [hunk 1]
```diff
@@ -1,42 +1,154 @@
...
-  Fonts
-*/
+/* Fonts
+==================== */
 
 @font-face {
     font-family: 'Fira Sans';
+    src: url('../fonts/FiraSans-Regular.eot');
+    src: url('../fonts/FiraSans-Regular.eot?#iefix') format('embedded-opentype'),
+         url('../fonts/FiraSans-Regular.woff') format('woff'),
```
Comment (akatsoulas) [inline feedthefox/base/static/css/app.css]: I would suggest to revert to the absolute path, as it was before.

Comment (akatsoulas) [inline feedthefox/base/static/css/app.css]: Please apply this to any occurrence in the code.

File: feedthefox/base/static/css/app.css [hunk 2]
```diff
@@ -1,42 +1,154 @@
...
 body {
     padding: 0;
     margin: 0;
-    backgroud-color: #eee;
-    color: black;
+    background-color: #eee;
+    color: #484848;
     font-size: 14px;
     line-height: 1.4;
     font-family: 'Fira Sans';
```
Comment (akatsoulas) [inline feedthefox/base/static/css/app.css]: Should we add a default font family here, eg 'sans-serif' ?

**Comment by comzeradd** [comment]
@akatsoulas thanks for the comments. PR updated.

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
 *.py[co]
 .env
 /static
+*.sqlite3
```
Comment (akatsoulas) [inline .gitignore]: Since we are not using this in the project, please remove it from here. You can always ignore this locally.

**Comment by akatsoulas** [comment]
What about using a sticky footer?

**Comment by akatsoulas** [comment]
Let's merge this and we can fix any issues that may arise as we proceed and have more specs/content. r+wc

**Comment by comzeradd** [comment]
Thanks. I fixed `.gitignore` and tweaked a bit the footer to stick at the bottom. We can revisit this when we have content.

**Pull request merged**